src/declarations/app-state.ts

```diff
@@ -0,0 +1,9 @@
+export class TxDetail {} // List of transactions. TODO: Handle memos, multiple addresses etc...
+
+export class Transaction {}
+
+export class RPCConfig {
+  username = ''
+  password = ''
+  url = ''
+}
```

Comment:
Declarations file is for `.d.ts` file only. This is a new type that has to be handled in the feature its related to

Reply:
I have moved to `.d.ts`